src/platform/auth/models/auth-tenant-user-response.dto.ts

```diff
@@ -6,12 +6,6 @@ import { AuthTenantResponseDto } from "./auth-tenant-response.dto";
 export class AuthTenantUserResponseDto {
   id!: string;
   role!: string;
-  email!: string;
```

Comment:
All these changes interfere with existing functionality which we must still support